content/docs/ops/runbook/rotating-secrets.md

```diff
@@ -0,0 +1,190 @@
+---
+menu:
+  docs:
+    parent: runbook
+
+title: Rotating Secrets I
+---
+
+## Introduction
+
+This guide assumes that you're following the Offboarding Checklist [found
+here](https://github.com/18F/cg-product/blob/master/OffboardingChecklist.md) and
+removed direct access to cloud.gov AWS accounts. The following guide walks a
+cloud.gov operator through rotating all sensitive information (secrets).
+
+Please familiarize yourself with [secret key management]({{< relref "docs/ops/secrets.md" >}}).
+You will be touching various parts of the platform from soup to nuts. Remember
+you have support from the entire team and it is helpful to coordinate with
+cloud.gov operators in case they are working on a particular piece of software
+that will requires a secrets rotation.
+
+## Rotate Access
+
+It's important that you rotate access to the platform first. This can be
+categorized with three different types of secrets.
+
+1. Remove access to Ops UAA
+1. [Rotate `cg-common` passphrases](#rotate-secrets-passphrases)
+1. [Rotate CF deployment users](#rotate-cf-deployment-users)
+1. [Rotate all IAM Users]({{< relref "docs/ops/runbook/rotating-iam-users.md" >}})
+1. [Rotating Bosh]({{< relref "docs/ops/runbook/rotating-bosh.md" >}})
+1. [Rotating Concourse]({{< relref "docs/ops/runbook/rotating-concourse.md" >}})
+1. [Rotating Cloud Foundry / Diego]({{< relref "docs/ops/runbook/rotating-cloudfoundry-diego.md" >}})
+
+## Rotate CF deployment users
+
+Some of our Concourse pipelines deploy to Cloud Foundry using cloud.gov service
+account deployers. In order to rotate these, you need to target the correct
+org and space in order to create the cloud.gov service deployer account
+instance.
+
+### Auditing all CF deployment users
```

Comment:
Can we drop this now that we're adding the shared broker deployer, or is it still useful?